core/storage/user/gae_models.py

```diff
@@ -2557,14 +2782,17 @@ def get_translation_reviewer_user_ids(cls, language_code):
             list(str). A list of IDs of users who have rights to review
             translations in the given language code.
         """
-        reviewer_keys = (
+        reviewer_keys = cast(
+            List[datastore_services.Key],
             cls.query(
                 cls.can_review_translation_for_language_codes == language_code)
-            .fetch(keys_only=True))
+            .fetch(keys_only=True)
```

Comment:
```suggestion
            ).fetch(keys_only=True)
```
and remove the bracket from prev line.

Reply:
Done